chain/network/src/ibf.rs

```diff
@@ -0,0 +1,185 @@
+use std::cmp::{max, min};
+use std::hash::Hasher;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::collections::hash_map::DefaultHasher;
+
+// const NUM_HASHES: usize = 3;
```

Comment:
should this be removed?

Reply:
We need this constant, I updated code to use it now.